### DISCUSSION TITLE
Setting up device mapping in docker with Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by olehomelchenko** [photoprism/photoprism#4167] [Q&A]
I run container based on the [docker-compose](https://dl.photoprism.app/docker/arm64/docker-compose.yml) for Raspberry Pi from the [documentation](https://docs.photoprism.app/getting-started/docker-compose/#__tabbed_1_3) on my Raspberry Pi 5.
In the devices block, there are following lines: 

```yaml
devices:
  - "/dev/video11:/dev/video11" # Video4Linux Video Encode Device (h264_v4l2m2m)
```

However, when I list contents of /dev directory on my device, I get no video11 but a list of other files: 

```sh
❯ ls /dev/ | grep video
video19
video20
video21
video22
video23
video24
video25
video26
video27
video28
video29
video30
video31
video32
video33
video34
video35
video36
video37
```

With this block commented out in docker-compose, everything runs ok but I wonder which of these files can I safely map to the container with Photoprism to enhance processing speeds for my photo library. 

Thanks in advance!

**Comment by inthreedee** [comment]
Late to the party, but I just ran into this upgrading from a Pi4 to a Pi5. Looks like they dropped hardware encoding in the Pi5:
https://forums.raspberrypi.com/viewtopic.php?p=2139082#p2139126

Supposedly they feel that the upgraded CPU is plenty fast to handle it better than the Pi4 even without hardware encoding:
https://www.raspberrypi.com/news/introducing-raspberry-pi-5/#comment-1594055

So, to answer the question, just comment out that line for the Pi5 and all is well.
